### DISCUSSION TITLE
How to use global css inside Ui folder!

### DISCUSSION THREAD

**Discussion by alifbinhossain** [vercel/turborepo#1943] [Help] [answered]
Hey, I am just a beginner exploring turbo. I have implemented tailwind css on this but still need some global styling for overall ui components. I need a global css file inside UI folder which can perform.  How can I achieve this! 
Please help me out!

**My current codebase looks like this..**

![Screenshot 2022-09-13 180038](https://user-images.githubusercontent.com/76260461/189895740-a2126e5c-6617-43cb-bbca-9302982979a7.png)

**Comment by anthonyshew** [comment, marked as answer]
Take a look at the Tailwind example! https://github.com/vercel/turbo/tree/main/examples/with-tailwind
